scaleway/provider.go

```diff
@@ -27,33 +27,38 @@ func Provider() terraform.ResourceProvider {
 		l.Errorf("cannot load configuration: %s", err)
 		return nil
 	}
+	p, err := scwConfig.GetActiveProfile()
```

Comment:
You have to manually get the varenvs with [scw.WithEnv](https://godoc.org/github.com/scaleway/scaleway-sdk-go/scw#WithEnv).